source/client/TableView.py

```diff
@@ -31,3 +42,51 @@ def Network_publicInfo(self, data):
         #    i) rerender immediately
         #    ii) rerender when explicitly told to
         # b) call Render with the provided data and only ever rerender on new broadcast
+
+    def playerByPlayer(self):
```

Comment:
This belong ABOVE the network comment dividing the class.

Reply:
Done.